api/src/pcapi/alembic/versions/20230712T150744_08dfd555fca2_add_categories_tags_offre_lieux.py

```diff
@@ -0,0 +1,49 @@
+"""
+add categories to offer and place tags
```

Comment:
(détail) Autant utiliser le nom de notre modèle. ;)

```suggestion
add categories to offer and venue tags
```